src/Metadata/AssemblyMetadata.Methods.cs

```diff
@@ -118,7 +118,11 @@ private EntityHandle ResolveMethodReference(MethodInfo method)
             }
 
             // Constructed generic method?
+#if NETSTANDARD || NET46
+            if (method.IsConstructedGenericMethod())
```

Comment:
See remark above: we should keep the extension method call for .NET Core as well and drop the `#if`.

Reply:
Done.